sycl/include/CL/sycl/accessor.hpp

```diff
@@ -382,24 +392,29 @@ class image_accessor
   };
 
   //  Available only when: (accessTarget == access::target::image ||
-  //  accessTarget == access::target::host_image) && accessMode ==
-  //  access::mode::read
+  //  accessTarget == access::target::host_image) && (accessMode ==
+  //  access::mode::read || accessMode == access::mode::read_write)
```

Comment:
I think the right condition is:
```c++
(accessTarget == access::target::image && accessMode == access::mode::read) || 
(accessTarget == access::target::host_image && (accessMode == access::mode::read || accessMode == access::mode::read_write))
```

Reply:
That is true. The static assert at line 284 takes care that read_write is not used with image targets.
So here the condition checked is as you have written.
Let me know if you still would like me to change this.

Reply:
Yes. I still think it should be fixed as this comment is misleading.